src/network/connection.cpp

```diff
@@ -298,6 +298,7 @@ void Connection::sendSensorInfo(Sensor& sensor) {
 	MUST(sendByte(sensor.getSensorId()));
 	MUST(sendByte(static_cast<uint8_t>(sensor.getSensorState())));
 	MUST(sendByte(static_cast<uint8_t>(sensor.getSensorType())));
+	MUST(sendByte(static_cast<uint8_t>(sensor.getMagStatus())));
```

Comment:
I don't like that it's sent here. We're adding flags and configs, but still expanding the sensorInfo packet. It should be sent with the rest of the config.

Reply:
I mean sure, but then think of the tracker initialization in the server also, do we defer it to the sensor flag list? This sensor flag list when do we send it? What are we gonna use it for aside for mag?

I feel like magnetometer fits well in the sensor info as it makes sense to have it there, but if we are gonna add a flag list I want it to make sense and used for more than just magnetometer so it's something that everyone can agree on (which will take a lot longer)

Reply:
Why do we need this information during initialization?

Reply:
because it doesnt make sense for a initialized tracker to be missing data? i can add a new enum for it on server called "WAITING" and stuff :sob:

aside from all that, how are we gonna make this work, do we just return a bitfield?